.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: npm
```

Comment:
Can we scope this to only a limited number of packages? If not, I think we can be flooded with updates for v8.. but I am not sure if this is going to work, as I expect that it will only look at the root dependencies. Does it support mono-repos and nested package.json files?

Reply:
Good questions.

I've took it from https://github.com/ghostery/adblocker/blob/master/.github/dependabot.yml
so it does support monorepos. 

Also as far I can can tell it cover root dependencies only.